ic-os/guestos/context/docker-base.dev

```diff
@@ -1 +1 @@
-ghcr.io/dfinity/guestos-base-dev@sha256:bd3cc2296731e78191dbf8a989b5809e102230de5fe509548e944632a97bd5cb
+ghcr.io/dfinity/guestos-base-dev@sha256:50d275df65c79bb216eb6f093df552e300220b309751187f3e71cd14c75c3e76
```

Comment:
This contains changes for the GuestOS 24.04 upgrade, we'll keep this in an RC for the initial rollout